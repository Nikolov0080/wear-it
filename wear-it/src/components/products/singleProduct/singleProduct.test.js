import React from 'react';
import renderer from 'react-test-renderer';
import SingleProduct from './singleProduct';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Single Product', () => {
    const tree = renderer
        .create(<BrowserRouter>
        <SingleProduct name='prodName' imageURL='klfsdkfjlsdf45345lkjlsdk'
        price="404598"
        id="_lkedf+fdpo" />
        </BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});