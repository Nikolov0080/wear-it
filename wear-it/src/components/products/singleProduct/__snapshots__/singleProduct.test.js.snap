// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Single Product 1`] = `
<div>
  <div
    className="card"
  >
    <a
      href="/details/prodName/_lkedf+fdpo"
      onClick={[Function]}
    >
      <img
        alt="product"
        className="image"
        src="klfsdkfjlsdf45345lkjlsdk"
      />
    </a>
    <div
      className="productInfo"
    >
      <h4
        className="productName"
      >
        prodName
      </h4>
      <h3
        className="productPrice"
      >
        404598
         USD
      </h3>
    </div>
    <div>
      <a
        className="link"
        href="/details/prodName/_lkedf+fdpo"
        onClick={[Function]}
      >
        <div
          as="div"
          className="add"
        >
          details
        </div>
      </a>
    </div>
  </div>
</div>
`;
